slides/2024/march/django-channels/index.html

```diff
@@ -0,0 +1,213 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
```

Comment:
Oh it's with reveal.js, Paul didn't manage to export it in PDF or images?

Reply:
You prefer it to be in PDF format ? I can try to export it.

Reply:
I think it's will be more easy if possible for people who don't want to run it in order to see the slides